engine/src/main/resources/org/camunda/bpm/engine/impl/mapping/entity/Authorization.xml

```diff
@@ -851,9 +856,13 @@
         </when>
         <otherwise>
           and (
-          (RES.CASE_EXECUTION_ID_ IS NOT NULL)
-          OR
-          (AUTH.RESOURCE_ID_ IS NOT NULL)
+          <!-- Task query does not use left join if cmmn is disabled, 
+            so we don't need to account for case tasks in this case. -->
+          <if test="authCheck.useLeftJoin" >
```

Comment:
Functionality-wise, this is absolutely fine. Naming-wise however, I am a bit concerned. 

Reasoning: 
Right now, we only use an authorization left join when CMMN is enabled. If we ever decide to have a left join here for any other reason, this assumption will not always be true. It is documented here, so people could definitely find it but it is easy to be missed as well.

As is said, I am just concerned for future updates. Other than that, it is fine.

Cleanest solution would probably be to introduce another property for that. But that might also be a bit too much.